manual_run/data_provider.py

```diff
@@ -0,0 +1,63 @@
+import csv
+from enum import Enum
+from os import path
+from typing import Dict
+
+import numpy as np
+
+from api.dataclasses.student import Student
+from benchmarking.data.interfaces import StudentProvider
+from manual_run.attributes import Attributes
+from manual_run.map_columns import process_row
+from manual_run.variables import Variables
+
+
+class DataProvider(StudentProvider):
+    def __init__(self):
+        self._max_project_preferences_per_student = 0
+
+    @property
+    def num_students(self) -> int:
+        return Variables.num_students
+
+    @property
+    def max_project_preferences_per_student(self) -> int:
+        return self._max_project_preferences_per_student
```

Comment:
I forgot to mention but this is also just always going to be 0 for now

Reply:
I believe we talked about this during our meeting and I think you may have just missed it because just above I have it set to always be zero.

<img width="698" alt="image" src="https://github.com/user-attachments/assets/7c625f97-4fd5-46c7-a164-31b20871ba3a">

Reply:
Ye I mean like there's no reason to have another variable just to set it to 0. Nothing is wrong, it's just simpler to have the method return 0 instead of return a variable that always equals 0